.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: ci
+on: pull_request
+jobs:
+  lint-commit:
+    name: lint (commit)
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - run: npm install --save-dev @commitlint/{config-conventional,cli} commitlint-plugin-jira-rules commitlint-config-jira
+      - name: Add dependencies for commitlint action
+        run: echo "NODE_PATH=$GITHUB_WORKSPACE/node_modules" >> $GITHUB_ENV
+      - run: git remote add upstream https://github.com/modin-project/unidist.git
+      - run: git fetch upstream
+      - run: npx commitlint --from upstream/master --to $(git log upstream/master..HEAD --pretty=format:"%h" | tail -1) --verbose
+
+  lint-black:
+    name: lint (black)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8.x"
```

Comment:
Why do we want to use 3.8.x instead of 3.7.x?

Reply:
3.8.x has longer period of support than 3.7.x. This frees us from updating the version of python for those checks in the future

Reply:
I believe that many people have still been working in env with python 3.7.x and testing their code with it. So I prefer to move to python 3.8.* with whole community.

Reply:
Including version 3.8.x in our linter checks we don't limit any way user to use others actual versions of python for developing. I prefer to leave it as it is.

Reply:
I checked `black` and `flack8` support py3 so there shouldn't be any issue with minor versions. Let's leave as it is.